src/components/picker/picker.tsx

```diff
@@ -26,6 +26,7 @@ import { createRandomString } from '../../util/random-string';
 const SEARCH_DEBOUNCE = 500;
```

Comment:
It feels like the changes to `menu-surface.tsx` and `portal.tsx` in this commit belong together (and might belong together with the first commit), while the changes to `picker.tsx` feel like they might belong in their own commit.

I assume the changes to `picker.tsx` are needed as a result of moving the dropdown height calculation from menu-surface to portal?

Reply:
yes, I agree 🙂

Reply:
:zap: